project/app/api/models.py

```diff
@@ -148,3 +146,49 @@ def check_valid_subid(cls, value):
     def check_sha_len(cls, value):
         assert len(value) == 128
         return value
+
+
+class LineGraphRequest(BaseModel):
+    """Model that handles LineGraph Request to API
+    """
+
+    ScoreHistory: List[int] = Field(..., example=[1005, 1500, 9000, 789])
+    StudentName: str = Field(..., example="Firstname")
+
+
+class HistogramRequest(BaseModel):
+    """Model that handles Histogram Request to API
+    """
+
+    GradeList: List[int] = Field(..., example=[1005, 1500, 9000, 789])
+    GradeLevel: int = Field(..., example=8)
+    StudentName: str = Field(..., example="Firstname")
+    StudentScore: int = Field(..., example=1058)
+
+
+class ClusterSubmission(BaseModel):
+    Image: str = Field(..., example="http://lorempixel.com/640/480/abstract")
+    Inappropriate: bool = Field(..., example=False)
+    Sensitive: bool = Field(..., example=False)
```

Comment:
Quick question: on the sensitive flag, since we don't actually have a wordlist & Graig said to opt for documentation rather than MVP implementation, will that break any functionality? Or will we just always be returning `False` for this until a new iteration is implemnted?

Reply:
Yeah it's on the todo list for features to come, for now it can just be returned as false without breaking anything.